packages/ui/views/welcome/welcome-step-three/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react'
+
+import { View, Text } from 'react-native'
+
+import style from 'theme/slide'
+
+import { mediumImage } from 'dimen/image'
+
+import { welcomeYouPlayWithFriends, welcomeTravelTogether, welcomeGreatConcert } from 'language'
```

Comment:
Once the previous `welcome` change is made, this becomes `import { welcome } from 'language'`